acct-user/motion/motion-0.ebuild

```diff
@@ -0,0 +1,13 @@
+# Copyright 2019-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit acct-user
+
+DESCRIPTION="added by portage for motion, a software motion detector"
+ACCT_USER_ID=395
+ACCT_USER_HOME=/var/lib/motion
```

Comment:
Is this actually needed? Please refer to 
http://michael.orlitzky.com/articles/gentoo_glep81_user_package_guidelines.xhtml#the-home-directory

Reply:
The reason for a home dir is that motion writes it's output files in the current working directory (or the directory defined in it's config file). Looking at Michael Orlizky's article I should create the home dir from the motion-video/motion ebuild though, not assume it'll be created by acct-user/motion ebuild.